src/common/shared.test.ts

```diff
@@ -24,10 +24,11 @@ THE SOFTWARE.
 
 /* eslint-disable import/first */
 window.require = require;
-
 import { updateAction } from "./shared";
 import { ActionContext } from "./types";
 
+window.require = require;
```

Comment:
Interesting - when first adding unit tests this only worked with the override before these imports.

We might want to looking into adding this globally since we're going to need it in many test files in the future.

Reply:
duplicated.